scripts/tokenomics-emulation/config.json

```diff
@@ -3,6 +3,7 @@
     "url": "wss://rpc.testnet.dev.cere.network:9945",
     "decimals": 10,
     "ddc_sc_address": "5FE4RyFK5NuK5GSkebp23fgJNGCg1imCsYTGjLQBQPybJQEU",
+    "cere_sc_address": "5C8VXSSmizvyUoKaTkyYTsqrchj3rAgjNHbNPJANhZkXqSfR",
```

Comment:
Is this address from DEV?

Reply:
No, This is for Prod.

Reply:
Update URL. Otherwise, you have inconsistency.

Reply:
Done